torch/_dynamo/eval_frame.py

```diff
@@ -512,13 +512,32 @@ def guard_export_print(guards):
 
 
 def export(
-    f, *args, aten_graph=False, decomposition_table=None, tracing_mode="real", **kwargs
+    f,
+    *args,
+    aten_graph=False,
+    decomposition_table=None,
+    tracing_mode="real",
+    aot_inductor=False,
```

Comment:
Does this need to be a flag?  IMO something like:
```
graph, guards = dynamo.export(...)
compiled = inductor.compile_fx(graph)
```
is easier to understand and hack on.   For example, you could run some custom passes before calling inductor.

Reply:
+1 - this should be fully after export